src/library/columncache.cpp

```diff
@@ -121,6 +121,8 @@ void ColumnCache::setColumns(const QStringList& columns) {
 
     insertColumnSortByEnum(COLUMN_TRACKLOCATIONSTABLE_LOCATION, kSortNoCase);
 
+    insertColumnSortByEnum(COLUMN_LIBRARYTABLE_COVERART, "CASE WHEN album_artist IS NULL THEN CASE WHEN GROUP BY album COUNT > 1 THEN \"Various Artists\" ELSE lower(artist) END ELSE lower(album_artist) END");
```

Comment:
Hard-coding an arbitrary, localized column value is not acceptable IMHO.

Reply:
Why not? Please explain and probably propose an alternative. I have already outlined details here: https://github.com/mixxxdj/mixxx/issues/11197#issuecomment-1382753549